DistributionPackages/Neos.NeosIo.ServiceOfferings/Resources/Private/JavaScript/ProviderListing.tsx

```diff
@@ -88,8 +99,8 @@ export default function ProviderListing() {
         <div>
             <div>
                 <header class="service-providers__grid-tableview">
-                    <div class="service-providers__grid-row remove-border form form--inline">
-                        <div class="service-providers__grid-cell">
+                    <div class="service-providers__grid-row remove-border form">
+                        <div class="service-providers__grid-cell row">
```

Comment:
Sadly the list view broke:

Now:
<img width="1167" alt="Bildschirmfoto 2023-09-28 um 10 11 32" src="https://github.com/neos/Neos.NeosIo/assets/596967/0fb527ce-d1d0-4a87-9bdd-9aad7681b918">

Before:
<img width="1174" alt="Bildschirmfoto 2023-09-28 um 10 11 41" src="https://github.com/neos/Neos.NeosIo/assets/596967/469e2951-e60f-4000-ba2a-9db88679505f">

Reply:
oh, thanks for watching! I didn't see that the classname was used in multiple cases and renamed the header class now to avoid interferences. It should look now as expected. :)

Reply:
Thx, will test it again today